plugins/woocommerce-admin/client/dashboard/components/connect/index.js

```diff
@@ -181,3 +174,16 @@ export default compose(
 		};
 	} )
 )( Connect );
+
+export const PrefetchJetpackAuthUrl = ( { redirectUrl = undefined } ) => {
```

Comment:
This component allows us to prefetch the Jetpack auth URL. Without prefetching, upon arriving at the task step where the "Connect" button is displayed, a user going through the task would have to wait until the REST API responds with a Jetpack URL which could take some time. With prefetching, there is no wait.

Reply:
Would it be possible to bring the prefetching to the `Setup` component as a `useEffect` hook? This would make it more consistent with the rest of WC Admin -- the `<PrefetchJetpackAuthUrl />` style component would be introducing a new pattern to the code base.

Reply:
Done in https://github.com/woocommerce/woocommerce/compare/3801bff79f7f5bab43d40c301d543078dcfa529b...de347bcffbef34a0a931911d00e9f29fa77edc6f